vendor/yubico/u2flib-server/.travis.yml

```diff
@@ -1,19 +1,21 @@
 language: php
 sudo: false
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
```

Comment:
Why is the Line 5.6 removed here when 5.6 should be allowed. 
It just blinked to my eye when I read through the changes. (Note that this commend can be absolutly idiotic, because I do not know the meaning of .travis.yml at all, just a hint to have a look to this again.
![screenshot_2](https://cloud.githubusercontent.com/assets/3265671/25842656/0d7d8abe-34a5-11e7-8ed2-5e5e34f4d4a3.png)
)